test/helper.rb

```diff
@@ -195,13 +195,11 @@ def md.directory?
       mock.mock_command('', '', '', 0)
     }
 
-    cmd_stderr = lambda { |x|
-      stderr = ::File.read(::File.join(scriptpath, '/unit/mock/cmd/'+x))
-      mock.mock_command('', '', stderr, 1)
+    cmd_exit_1 = lambda { |x = nil|
+      stderr = ::File.read(::File.join(scriptpath, 'unit/mock/cmd', x)) if x
```

Comment:
You might think I'm nitpicking here, but I think we can eliminate the conditional in the following line and just let `stderr` always be the right thing (rather than `mock_command` having another in-line conditional):

```ruby
stderr = x.nil? ? '' : ::File.read(::File.join(scriptpath, 'unit/mock/cmd', x))
mock.mock_command('', '', stderr, 1)
```

That's a bit more direct and clear, would you agree?

Reply:
Yup, I agree, elegant use of the ternary. Not nitpicky at all! 😄